src/main/java/com/codegym/jrugotom5/controller/AdvertController.java

```diff
@@ -39,4 +40,26 @@ public List<AdvertBasicInfoDTO> getByCategory(@PathVariable String category) {
         return advertService.getByCategory(category);
 
     }
+
+    @DeleteMapping("/advert/{id}")
```

Comment:
/{id} is enough since controller class is mapped with /api/adverts

Reply:
changes have been made